go.mod

```diff
@@ -3,12 +3,16 @@ module sigs.k8s.io/cli-utils
 go 1.16
 
 require (
+	github.com/google/go-cmp v0.5.6
 	github.com/google/uuid v1.2.0
+	github.com/mikefarah/yq/v4 v4.12.1
 	github.com/onsi/ginkgo v1.16.2
 	github.com/onsi/gomega v1.12.0
+	github.com/pkg/errors v0.9.1
```

Comment:
Please don't add a dependency on this package as it's not really necessary. I've removed it recently in https://github.com/kubernetes-sigs/cli-utils/pull/380

Reply:
Gotcha. Will remove.

Reply:
Thanks!